### DISCUSSION TITLE
Ory Elements: How to control order and UI width?

### DISCUSSION THREAD

**Discussion by hyperknot** [ory/.github#44] [Q&A] [answered]
In Elements, how can I control the order between SSO and username & password? I'd like to have SSO in the bottom and username in the top part.

Also, is there a way to set the UI component's width? From what I see the width is fixed and there is no CSS variable exported for that.

**Comment by hyperknot** [comment]
The only workaround I found for the UI width, is to use a class which overwrites max-width. In Tailwind the following class can set the width:

`className="w-96 !max-w-none"`.

**Comment by hyperknot** [comment]
As I see the order of OIDC and password is very much hard coded. Please take it as a feature request to reverse that order.

**Comment by aeneasr** [comment]
@Benehiko does it make sense to move this into an issue?

**Reply by Benehiko** [reply]
Yes, let's take this to an issue since we will need to add custom ordering.

**Comment by Benehiko** [comment, marked as answer]
Please see this issue 
https://github.com/ory/elements/issues/84
